### DISCUSSION TITLE
Default LazyVim with tokyonight and undercurls do not show up in LSP diagnostics

### DISCUSSION THREAD

**Discussion by scottbenjamin** [LazyVim/LazyVim#474] [Q&A]
Hi!

I am trying to track down why undercurls don't show up even though my terminal supports them and works. From looking at tokyonight github readme it's supported.

Is there some set of configuration in LazyVim which disables them?

I just tried checking out LazyVim starter and used those configs to confirm it doesn't work with 100% default configuration as well.

Thanks in advance.

**Comment by folke** [comment]
nope, if your terminal supports it, it should work. nothing to configure. Make sure your terminal has proper termcap installed and if you're using tmux, you need additional configuration. I don't use tmux myself, so can't help you withthat
